Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM openjdk:8-jre-alpine
-ADD target/scala-2.13/hello-urld-assembly-0.1.0.jar app.jar
-ENTRYPOINT ["java", "-jar", "/app.jar"]
\ No newline at end of file
+FROM mozilla/sbt
```

Comment:
Luckily, [Mozilla already had a Dockerfile with sbt](https://github.com/mozilla/docker-sbt) so I chose this for an easy demo setup instead of building the fat .jar file or [setting up Play's production mode](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-slick-example).